crawler.py

```diff
@@ -70,6 +70,9 @@
 ap.add_argument('--log-stdout', action='store_true', default=False,
                 help='If set, log to stdout as well as log.txt')
 
+ap.add_argument('--load-extension', default=None,
```

Comment:
We should log the value of `--load-extension` alongside all the other significant crawl settings.

Reply:
Good call :+1: added this